services/provisioningservices/mgmt/2017-08-21-preview/iothub/version.go

```diff
@@ -21,10 +19,10 @@ import "github.com/Azure/azure-sdk-for-go/version"
 
 // UserAgent returns the UserAgent string to use when sending http.Requests.
 func UserAgent() string {
-	return "Azure-SDK-For-Go/" + version.Number + " iothub/2017-08-21-preview"
+	return "Azure-SDK-For-Go/latest services"
 }
 
 // Version returns the semantic version (see http://semver.org) of the client.
 func Version() string {
-	return version.Number
```

Comment:
@jhendrixMSFT, here's an example of of something that doesn't seem to be picking up the `use-onever` flag for some reason. [Could you verify that I have typed the name of that flag correctly here? ](https://github.com/Azure/azure-sdk-for-go/blob/01ff0b9b4ce9bfe37e983feabc301ebfac7db745/swagger_to_sdk_config.json#L12)

Reply:
The flag is correct, and from the codegen this generator is recent and so the support for the flag is present.  Is there a log file from the AutoRest invocation?